### ISSUE TITLE
Accessibility issue: Unable to select the images while accessibility on in Android/IOS

### ISSUE THREAD

**Issue by ArjunPatidarRadix** [ivpusic/react-native-image-crop-picker#1965]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.40.0
- react-native v0.72.4

### Platform
Tell us to which platform this issue is related

- iOS
- Android

### Expected behaviour
It should select the photos from the gallery on double tap while accessibility on

### Actual behaviour
I'm making my app accessible with voice over IOS and talk android, But when the ImagePicker.openPicker({ and I'm unable to access and select the photos.

### Steps to reproduce

1. Turn on voice-over in IOS from the setting

2. call ImagePicker.openPicker({

3. Try to select the images on a double-tap

### Attachments

// stacktrace or any other useful debug info

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by Nanjundan** [comment]
@ArjunPatidarRadix ArjunPatidarRadix Did you find any solution for this?
I am also facing the issue where images are not read by voice over and unable to select using double tap.

**Comment by Nanjundan** [comment]
https://github.com/ivpusic/react-native-image-crop-picker/pull/1439
This PR merge will help you on the issue
